internal/store/pod.go

```diff
@@ -860,6 +860,64 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_pod_init_container_resource_requests",
```

Comment:
how come this is part of this PR?

Reply:
Because we documented `kube_pod_init_container_resource_requests` in the doc but we forgot for collect it in pod.go.  I have also changed the way we collect tested metrics. `kube_pod_init_container_resource_requests` is just added in the `metadata` field of Prometheus metrics output instead really collect it and contains it in the ouput.

Reply:
@andyxning Thanks for catching this. Looks like a bad metric has been added: `"kube_init_pod_container_resource_requests_cpu_cores"`

Can you remove that completely? CPU core requests are deprecated for pod containers and should never be added to pod init containers since they are new.